cypress-tests/cypress/e2e/PaymentTest/00013-SaveCardFlow.cy.js

```diff
@@ -37,7 +37,7 @@ describe("Card - SaveCard payment flow test", () => {
           "card_pm"
         ]["SaveCardUseNo3DSAutoCapture"];
         let req_data = data["Request"];
-        let res_data = data["Response"];
+        let res_data = data["ResponseCustom"] ? data["ResponseCustom"] : data["Response"];
```

Comment:
I feel this wont work as what we expect.

We, at least, need to make this depend on the response status, not a boolean value while giving priority to `Response` and not `ResponseCustom`. 

Main use case of using `ResponseCustom` is that when you're expecting the connector to throw 2 or more different responses for a payment.